src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -26,10 +28,17 @@ class ArrayField extends React.Component {
     super(props);
     // In contrast to the normal array field, we don’t want to add an empty item
     // and always show at least one item on the review page
-    const arrayData = Array.isArray(props.arrayData) ? props.arrayData : null;
+    const arrayData = (Array.isArray(props.arrayData) && props.arrayData) || [];
+
+    // Including an `duplicateKey` is what causes this unique entry validation to
+    // open the array card in edit mode
+    const key = props.uiSchema?.['ui:options']?.duplicateKey || '';
```

Comment:
Yeah, that's good. :+1: 

I wish we had a better place to document this, though. :weary: 

```suggestion
    // Including a `duplicateKey` is what causes this unique entry validation to
    // open the array card in edit mode
    const key = props.uiSchema?.['ui:options']?.duplicateKey || '';
```

Reply:
Yeah... maybe we could start building JSDocs like Jeff B is doing?